pkg/executor/bacalhau/bacalhau.go

```diff
@@ -49,18 +49,20 @@ func (executor *BacalhauExecutor) Id() (string, error) {
 	)
 	nodeIdCmd.Env = executor.bacalhauEnv
 
-	output, err := nodeIdCmd.CombinedOutput()
+	runOutputRaw, err := nodeIdCmd.CombinedOutput()
 	if err != nil {
 		return "", fmt.Errorf("error calling get id results %s", err.Error())
 	}
 
+	splitOutputs := strings.Split(strings.Trim(string(runOutputRaw), " \t\n"), "\n")
+	runOutput := splitOutputs[len(splitOutputs)-1]
```

Comment:
```go
errOutput = splitOutputs[1:]
if errOutput != nil {
  return "", fmt.Errorf("error unmarshalling job JSON %s %s", err.Error(), errOutput)
}
```

Reply:
but in correct situation,  [1:] may panic
![image](https://github.com/Lilypad-Tech/lilypad/assets/41407352/7839907b-5746-494a-b713-40c3763c4c1c)

Reply:
![image](https://github.com/Lilypad-Tech/lilypad/assets/41407352/38efa30d-00b3-46d8-9009-b0c834d22c7e)
add like this, but i got a panic in this change. i dont know the how the error log look like

Reply:
If you are getting errors, it means there is something wrong, you also need to trim, I added a similar change recently, here: https://github.com/Lilypad-Tech/lilypad/pull/175/files
Let's make sure we also capture errors if there are errors

Reply:
@AquiGorka I've updated to add logging for error output.
@hunjixin Looks like we want `splitOutputs[:len(splitOutputs)-1]` (i.e. everything but the last line).

Reply:
this change seem to make things not work.the output in my comment 
1 line: error log
1 line: id result
1 line: empty line

this is why add a trim to make sure the empty line is removed